metrics/github/cli.py

```diff
@@ -100,11 +100,9 @@ def github(ctx):
         prs = list(api.iter_prs(org))
         log.info("Backfilling with %s PRs for %s", len(prs), org)
 
-        rows = list(
-            itertools.chain(
-                old_prs(prs, org, days_threshold=7),
-                pr_throughput(prs, org),
-            )
+        rows = itertools.chain(
+            old_prs(prs, org, days_threshold=7),
+            pr_throughput(prs, org),
         )
```

Comment:
You could even drop the `itertools` import...

```suggestion
        rows = (
            pr
            for group in [old_prs(prs, org, days_threshold=7), pr_throughput(prs, org)]
            for pr in group
        )
```

Reply:
Good point about dropping `itertools`. I think there might be an even better solution that might be easier to read, which is to write the results of each function call to the database separately. Anyway, I won't make any further changes in this PR, but I do think this is worth doing in future.